src/main/java/com/investmenttracker/enterprise/service/IInvestmentService.java

```diff
@@ -1,27 +1,27 @@
 package com.investmenttracker.enterprise.service;
 
 import com.investmenttracker.enterprise.dto.MarketData;
-import com.investmenttracker.enterprise.dto.investment;
+import com.investmenttracker.enterprise.dto.Investment;
 
 import java.io.IOException;
 import java.util.List;
 
 public interface IInvestmentService {
-    investment fetchById(int id);
+    Investment fetchById(int id);
 
     void delete(int id) throws Exception;
 
-    void save (investment Investment);
+    void save (Investment investment);
```

Comment:
Variable names should be lowercase for the first letter. I refactor a lot of these as well throughout the whole code base. https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3